common/src/api/external/mod.rs

```diff
@@ -627,6 +627,7 @@ impl Generation {
         Generation(1)
     }
 
+    #[must_use]
     pub fn next(&self) -> Generation {
```

Comment:
I hit this issue locally, thinking that this was acting on `&mut self`, so I figured I'd the annotation to guard against a similar mistake.